qubes-castable/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use core::mem::size_of;
 #[macro_export]
 macro_rules! static_assert {
     ($e: expr) => {{
-        let [] = [0; if $e { 0 } else { 1 }];
+        const _: () = assert!($e);
     }};
 }
```

Comment:
You could accept format arguments within `static_assert!` to avoid reimplementing it below. Though right now it's simple enough that not using the macro is also sensible.